### DISCUSSION TITLE
where to download

### DISCUSSION THREAD

**Discussion by keiron26** [raspicamplayer/camplayer#88] [Camplayer OS]
Hi i want to download the camplayer os but the link dont seem to work keeps saying the file set to privet 

Thanks 
K

**Comment by jonboy86** [comment]
i have the same issue

**Comment by Adicke** [comment]
I add myself to the issue.  
The OS would be nice to have.

**Comment by webdatatech** [comment]
Anyone have a copy of the OS they downloaded?

**Comment by evousa** [comment]
I want to download OS , but I don't now how do that ! Someone can help me with that ?
